inc/broadcast/namespace.php

```diff
@@ -10,9 +10,12 @@
 use Altis\Accelerate\Admin;
 use Altis\Analytics\API;
 use Altis\Analytics\Utils;
+
+use function Altis\Analytics\API\get_block_preview_thumbnail;
```

Comment:
Avoid use function in favour of putting the last part of the namespace before the function